tutorial/westeros/westeros_baseline.ipynb

```diff
@@ -1338,9 +1325,9 @@
    "source": [
     "### Electricity Price\n",
     "\n",
-    "And how much does the electricity cost? These prices are taken from the **dual variables** of the solution and are given the name **shadow prices**. They reflect the marginal price of electricity, taken from the most expensive producer.  \n",
+    "And how much does the electricity cost? These prices are in fact **shadow prices** taken from the **dual variables** of the model solution. They reflect the marginal cost of electricity generation (i.e., the additional cost of the system for supplying one more unit of electricity), which is in fact the marginal cost of the most expensive generator.  \n",
```

Comment:
Thanks for this. I clarified as "... of the most expensive operating generator”. Because here we only have two technologies and both are in use. Sometimes, a cheap generator can have unused capacity that hasn't been active due to other constraints than cost (e.g., growth rates).

Reply:
> Sometimes, a cheap generator can have unused capacity that hasn't been active due to other constraints than cost (e.g., growth rates).

Ah, true.